admin_scripts/hooks/protect_config_files.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+#================================================================
+# HEADER
+#================================================================
+#% SYNOPSIS
+#+    protect_config_files.sh
+#%
+#% DESCRIPTION
+#%    This script installs or removes pre- and post-checkin hooks that
+#%    together prevent OS2borgerPC configuration files from being modified.
+#%
+#%    It is only useful as an extra safeguard during certain remote upgrades.
+#%
+#================================================================
+#- IMPLEMENTATION
+#-    version         protect_config_files.sh (magenta.dk) 1.0.0
+#-    author          Alexander Faithfull
+#-    copyright       Copyright 2021, Magenta ApS
+#-    license         GNU General Public License
+#-    email           af@magenta.dk
+#-
+#================================================================
+#  HISTORY
+#     2021/03/15 : af : Script created
+#
+#================================================================
+# END_OF_HEADER
+#================================================================
+
+set -x
+
+if [ ! -f "/usr/local/bin/jobmanager.real" ]; then
+    echo "This machine does not have jobmanager hook support"
+    exit 1
+fi
+
+HOOKS="/etc/os2borgerpc/pre-checkin.d/protect_config_files.sh
+/etc/os2borgerpc/post-checkin.d/protect_config_files.sh"
+
+if [ "$1" != "" -a "$1" != "false" -a "$1" != "falsk" ]; then
```

Comment:
Som ovenfor.

Reply:
Det ser fint og helt rigtigt ud bortset fra min lille kommentar om ja/nej/true/false-protokollen for aktivering.